data/RedHat-family.yaml

```diff
@@ -26,7 +26,7 @@ galera::mariadb::103::galera_package_name: 'galera'
 galera::mariadb::103::mysql_package_name: 'MariaDB-server'
 galera::mariadb::bootstrap_command: '/usr/bin/galera_new_cluster'
 galera::mariadb::libgalera_location: '/usr/lib64/galera/libgalera_smm.so'
-galera::mariadb::mysql_service_name: 'mysql'
+galera::mariadb::mysql_service_name: 'mariadb'
```

Comment:
Is this true across all supported releases of RedHat/CentOS – 6,7,8 and different versions of MariaDB? I don't want to risk breaking existing setups on older platforms.

Reply:
I've checked puppetlabs-mysql and it seems this is also true for old versions.